gitbook-docs/rest_api.md

```diff
@@ -48,15 +44,24 @@ of the specification and there is no registry for id values. If providfed, the `
 as `swname` and `swvers` within the [DNS-SD advertisement](connection.md) (if implemented), and also the `id` MUST
 provide the same value as `name` within the [Websocket hello message](streaming_api.md) (if implemented).
 
-## /signalk/«version»/api/
+# v1 specification
+
+If a REST API is provided it MUST be located at `http[s]://<<host>>:[port]/signalk/v1/api/`
+If a streaming websocket is provided it MUST be located at `ws[s]://<<host>>:[port]/signalk/v1/stream`
+
+The discovery json document published by an IPv4 address should advertise IPv4 endpoints and similarly the discovery json at an IPv6 address should advertise IPv6 endpoints.
```

Comment:
I don't think this should be stated in the spec. If a server wants it can advertise IPv6 over IPv4 - dunno how much sense it makes, but nevertheless I don't think the spec should make restrictions here.